tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -48,14 +51,39 @@ void ROOT::RDF::RunGraphs(std::vector<RResultHandle> handles)
    std::set<RResultHandle, decltype(sameGraph)> s(handles.begin(), handles.end(), sameGraph);
    std::vector<RResultHandle> uniqueLoops(s.begin(), s.end());
 
+   // Trigger jitting. One call is enough to jit the code required by all computation graphs.
+   TStopwatch sw;
+   sw.Start();
+   {
+      auto silenceRDFLogs = ROOT::Experimental::RLogScopedVerbosity(ROOT::Detail::RDF::RDFLogChannel(),
+                                                                    ROOT::Experimental::ELogLevel::kError);
+      uniqueLoops[0].fLoopManager->Jit();
```

Comment:
This relies on the fact that all the different `RLoopManagers` will share a common code to be jitted, right?

Reply:
Also, does this mean that at this time we are re-jitting once per unique `RLoopManager` ?

Reply:
> all the different RLoopManagers will share a common code to be jitted, right?

that's currently the case.

> at this time we are re-jitting once per unique RLoopManager ?

no, code to jit is accumulated globally for all computation graphs, and it is jitted and executed the first time anyone needs it